assembl/static2/js/app/components/debate/thread/threadView.jsx

```diff
@@ -29,10 +29,24 @@ type Props = {
 };
 
 class ThreadView extends React.Component<Props> {
+  // failed to type it React$Ref<HTMLDivElement>
+  threadViewRef: any;
+
   static defaultProps = {
     postsDisplayPolicy: defaultDisplayPolicy
   };
 
+  componentWillMount(): void {
+    this.threadViewRef = React.createRef();
+  }
+
+  scrollToTop = () => {
+    window.scrollTo({
+      top: this.threadViewRef.current.getBoundingClientRect().y + window.pageYOffset - 160,
```

Comment:
What's the 160px for? Just a random number or size of navigator? Perhaps can make into a constant